iconik-storage-gateway.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "iconik-storage-gateway",
-  "release": "11.2-RELEASE",
-  "artifact": "https://github.com/Cantemo/iocage-plugin-iconik-storage-gateway.git",
+  "release": "12.2-RELEASE",
+  "artifact": "https://github.com/iconik-io/iocage-plugin-iconik-storage-gateway.git",
   "properties": {
     "vnet":"on"
   },
   "pkgs": [
-    "lang/python36",
+    "lang/python38",
```

Comment:
iirc these changes are just limited here and for end users they are not propagated. Can you please confirm if you would like these changes made to freenas/iocage-ix-plugins repository ?

Reply:
I might not understand fully what you mean. In the iocage-ix-plugins the manifest points to this repo as the "artifact", so I would assume it is used on jail creation, right?
I upgraded the python version just because pyrhon36 is getting deprecated soon, the manager that edits the config uses it, ISG is shipped with its own frozen python env which is python38 now.

Reply:
@nech we should be making the changes for the manifest in `freenas/iocage-ix-plugins`as the json manifest located there tells the system which pkgs should be installed or which artifact should be used. Also if we want to change the artifact repository, then rest of the changes like LICENSE changes should also go to the new repository.

Reply:
So for example, if we want to use the repo `https://github.com/iconik-io/iocage-plugin-iconik-storage-gateway.git`, no changes are required in this repository and all the changes should go to `https://github.com/iconik-io/iocage-plugin-iconik-storage-gateway.git` apart from the manifest changes. They would still go to `freenas/iocage-ix-plugins`

Reply:
Got it, thank you! I assumed at that moment it also uses manifest from the artifact installing all of the dependencies, but now I understand that they are part of the `iconik_storage_gateway` package hosted on `http://pkg.cdn.trueos.org/iocage/unstable`.

The `https://github.com/iconik-io/iocage-plugin-iconik-storage-gateway.git` repo fetches the iconik_storage_gateway package from the cloud bucket where we host it. If it's not a problem to install it this way we could use it. Then we can cancel this PR and I will update the manifest file in the PR I already created in `freenas/iocage-ix-plugins`.

If you want to continue with the old workflow, what would be the next steps? The current manifest file in `freenas/iocage-plugin-iconik-storage-gateway` points to the artifact repo that is no longer present, so I guess we should still add these changes? But not the ones in `post_install.sh` since the package will be installed from `http://pkg.cdn.trueos.org/iocage/unstable` instead.

Reply:
@nech we can use `https://github.com/iconik-io/iocage-plugin-iconik-storage-gateway.git` repository for the artifact which defines post installation logic and other bits. Then we can update `freenas/iocage-ix-plugins` to make sure iconik manifest reflects the new reality.
Please let me know if there are concerns or if you would like me to clarify any bit. Thanks!

Reply:
Ok. I'll close this PR and create a new one.